### PR TITLE
adding new parameters as per PR on the code

### PR DESCRIPTION
**Pull request by wasperen**
## Proposed change

These changes describe how the additional parameters (unlocked through https://github.com/wasperen/hass-core/commit/d7825c31b4fb97d55964f488da2f9615867943ba) can be used.

Also made it clear that `client_id` is actually called "Client key" on the current version of the Mastadon UI.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/wasperen/hass-core/commit/d7825c31b4fb97d55964f488da2f9615867943ba

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/mastodon.markdown [hunk 1]
```diff
@@ -17,7 +17,12 @@ The `mastodon` platform uses [Mastodon](https://joinmastodon.org/) to deliver no
 
 ### Setup
 
-Go to **Preferences** in the Mastodon web interface, then to **Development** and create a new application.
+Go to **Preferences** in the Mastodon web interface, then to **Development** and create a new application. This
+integration currently only support sending notifications, so the API only has to be able to `write` to the
```
Comment (c0ffeeca7) [inline source/_integrations/mastodon.markdown:21]:
```suggestion
integration currently only supports sending notifications, so the API only has to be able to `write` to the
```

File: source/_integrations/mastodon.markdown [hunk 2]
```diff
@@ -17,7 +17,12 @@ The `mastodon` platform uses [Mastodon](https://joinmastodon.org/) to deliver no
...
+account.
+
+NB: although the parameter is called `client_id`, the web front-end of (at least) version 4.0.2 calls this parameter
+Client key.
```
Comment (c0ffeeca7) [inline source/_integrations/mastodon.markdown:25]:
```suggestion
**Client key**.
```

File: source/_integrations/mastodon.markdown [hunk 3]
```diff
@@ -17,7 +17,12 @@ The `mastodon` platform uses [Mastodon](https://joinmastodon.org/) to deliver no
 
 ### Setup
 
-Go to **Preferences** in the Mastodon web interface, then to **Development** and create a new application.
+Go to **Preferences** in the Mastodon web interface, then to **Development** and create a new application. This
+integration currently only support sending notifications, so the API only has to be able to `write` to the
+account.
+
```
Comment (c0ffeeca7) [inline source/_integrations/mastodon.markdown:23]: Do you think you could maybe also specify what to put for **Application website** and **Redirect URI** on the Mastodon end of things? If nothing is required there, maybe just say that it's fine to leave those fields empty. I tried following your steps and felt uncertain there.

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/mastodon.markdown [hunk 4]
```diff
@@ -59,3 +64,48 @@ base_url:
...
+{% configuration %}
+message:
+  description: "The message to be sent, potentially including the full handles of referenced other Mastodon users."
+  required: true
+  type: string
+  default: ""
+visibility:
+  description: "determines the scope of how the Toot is going to be sent. Can be any of: ‘direct’ - post will be 
+visible only to mentioned users; ‘private’ - post will be visible only to followers; ‘unlisted’ - post will be 
+public but not appear on the public timeline; ‘public’ - post will be public (the default)"
```
Comment (frenck) [inline source/_integrations/mastodon.markdown:97]:
```suggestion
  description: "determines the scope of how the Toot is going to be sent. Can be any of: `direct` - post will be 
visible only to mentioned users; `private` - post will be visible only to followers; `unlisted` - post will be 
public but not appear on the public timeline; ‘public’ - post will be public (the default)"
```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @wasperen 👋 

Thanks for the PR; however, it seems like the PR is targeting an incorrect branch. Currently, it is targeting the `current` branch, however, it depends on a code change in the Core repository, which will be released in a future (next) version of Home Assistant. Therefore, this PR should target the `next` branch.

Please change the target to `next` and git rebase your changes if needed.

Thanks 👍 

../Frenck

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**